### DISCUSSION TITLE
Best browserslist config for capacitor 3 ?

### DISCUSSION THREAD

**Discussion by ThomasKientz** [ionic-team/capacitor#4327] [Q&A]
Which browserslist config is best suited to target maximum devices when bundling (eg https://cli.vuejs.org/guide/browser-compatibility.html#browserslist) to production and ship it with capacitor 3 ? 

[Capacitor 3 now builds for ES2017 environments](https://capacitorjs.com/docs/v3/updating/3-0#es2017)
[Capacitor 3 supports Android 5+](https://capacitorjs.com/docs/v3/updating/3-0#ios)
[Capacitor 3 supports iOS 12+](https://capacitorjs.com/docs/v3/updating/3-0#ios)

**Comment by jcesarmobile** [comment]
Capacitor 3 supports Android WebView/Chromium version 60 or greater, not sure if that helps. I'm not very familiar with browserlist.
